tests/scanner/test_data/fake_enabled_apis_scanner_data.py

```diff
@@ -18,24 +18,30 @@
 
 
 API_COMPUTE = {
-    'serviceName': 'compute.googleapis.com',
-    'producerProjectId': 'compute-producer-project'}
+    "config": {
```

Comment:
By doing this, will the `servicemanagement` be untested?  Should these fake data be in a separate file, specifically for `fake_serviceusage_data`?

Reply:
I don't think servicemanagement being untested is an issue, since we are not using that API for this scanner, and there is a separate file for running ServiceManagement API tests.